lib/mayaUsd/render/mayaToHydra/utils.h

```diff
@@ -45,6 +50,31 @@ using MtohRendererDescriptionVector = std::vector<MtohRendererDescription>;
 using MtohRendererSettings
     = std::unordered_map<TfToken, HdRenderSettingDescriptorList, TfToken::HashFunctor>;
 
+// Defining these in header so don't need to link to use... template is
+inline bool IsMtohRenderOverrideName(const MString& overrideName)
+{
+    // See if the override is an mtoh one - ie, it starts with the right prefix
+    constexpr auto prefixLen = strlen(MTOH_RENDER_OVERRIDE_PREFIX);
+
+    TF_WARN(
+        "%s: %s an mtoh renderer",
+        overrideName.asChar(),
+        overrideName.substring(0, prefixLen - 1) == MTOH_RENDER_OVERRIDE_PREFIX ? "is" : "is not");
+
+    if (overrideName.length() < prefixLen) {
+        return false;
+    }
```

Comment:
If the view is not using any render override, do we return from this check?

Reply:
Yes - `overrideName` is the empty string in that case